cmd/dbaas-controller/main.go

```diff
@@ -26,7 +26,8 @@ import (
 	"google.golang.org/grpc/grpclog"
 	"gopkg.in/alecthomas/kingpin.v2"
 
-	_ "github.com/percona-platform/dbaas-controller/catalog" // load messages
+	// load messages.
+	_ "github.com/percona-platform/dbaas-controller/catalog"
```

Comment:
No, that comment should be on that line – it explains the blank import

Reply:
Yeah, it was incorrectly formatted by `golangci-lint run --fix`

Reply:
fixed.